relay-metrics/src/view.rs

```diff
@@ -744,15 +745,15 @@ enum SplitDecision {
 /// Calculates a split for this bucket if its estimated serialization size exceeds a threshold.
 ///
 /// There are three possible return values:
-///  - `BucketFits` if the bucket fits entirely into the size budget.
+///  - `BucketFits(size)` if the bucket fits entirely into the budget and consumes `size` bytes.
 ///  - `MoveToNextBatch` if the size budget cannot even hold the bucket name and tags. There is no
 ///    split, the entire bucket is moved.
-///  - `Split(at)` if the bucket fits partially, the bucket should be split `at`.
+///  - `Split(n)` if the bucket fits partially, the bucket should be split after `n` elements.
```

Comment:
This change in documentation along with the remaining renaming should help to avoid this mistake going forward.